### PR TITLE
Support creating sequence before trasnfer the table data

### PR DESCRIPTION
**Pull request by baotingfang**
* Create the sequence which is belonged to a table.
* Correct the sequence next value

We will check the dependent sequences before transfering the table data.
If the table contains the sequeces. gptransfer will pg_dump all the
metadata of the sequences with data. and import it into the
destiantion cluster

Signed-off-by: Ming LI <mli@apache.org>

### CODE REVIEW

File: gpMgmt/bin/gptransfer [hunk 1]
```diff
@@ -1887,6 +1893,72 @@ FROM (
...
+            sequence_sql = cmd.get_schema_sql()
+            if sequence_sql:
+                _, temp_seq_file = tempfile.mkstemp(suffix='.sql')
+
+                with open(temp_seq_file, 'w') as f:
+                    f.write(sequence_sql)
+
+                # In case of --full option, sequence will be created at the second time
+                # So here execute sql (e.g. setval()) with psql even if the sequence exists.
+                psql_cmd = PsqlFile('Create full schema',
```
Comment (adam8157) [inline gpMgmt/bin/gptransfer]: "Create full schema"?

File: gpMgmt/bin/gptransfer [hunk 2]
```diff
@@ -1718,6 +1723,7 @@ class GpTransferCommand(Command):
                             self._table_pair.dest.database, self._dest_user)
                 self._dest_conn = connect(url)
 
+                self._check_sequences()
```
Comment (adam8157) [inline gpMgmt/bin/gptransfer]: Not only "check" but also create, how about renaming this to "_transfer_sequences()"?

File: gpMgmt/bin/gptransfer [hunk 3]
```diff
@@ -1887,6 +1893,72 @@ FROM (
...
+                    self._table_pair.source)
+            cmd = GpSchemaDump(
+                'schema dump of %s' % self._table_pair.source,
+                self._src_host,
+                self._src_port,
+                self._src_user,
+                False,
+                self._table_pair.source.database,
+                seq.schema,
+                seq.table,
```
Comment (adam8157) [inline gpMgmt/bin/gptransfer]: Better to add some comments here, "seq.table" looks like transferring the table, but it's only the sequence.

Comment (baotingfang) [reply gpMgmt/bin/gptransfer]: Good Idea. Adding more comments is better!

File: gpMgmt/bin/gptransfer (hunk 1 above)
Comment (baotingfang) [reply gpMgmt/bin/gptransfer]: Just dump the sequence metadata and data.

**Pull request merged**